src/Apps/Feature/Components/FeatureHeader/FeatureHeader.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react"
+import styled from "styled-components"
+import { flexBasis } from "styled-system"
+import { Flex, Sans, color, Image, Spacer } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { FeatureHeader_feature } from "__generated__/FeatureHeader_feature.graphql"
+
+const Container = styled(Flex)`
+  height: 100%;
+  width: 100%;
+  flex: 1;
+`
+
+const Figure = styled(Flex)`
+  flex-basis: 50%;
+  flex-shrink: 0;
+  flex-grow: 0;
+  background-color: ${color("black10")};
+`
+
+const Cover = styled(Image)`
+  object-fit: cover;
+  width: 100%;
+  height: 100%;
+`
+
+const Meta = styled(Flex)`
+  ${flexBasis}
```

Comment:
Weird that we missed this function in the Flex component. Will be nice to just upgrade styled-system and use the `flexbox` function.

Reply:
Yah, will do.